### ISSUE TITLE
Re wrap of the mdbook

### ISSUE THREAD

**Issue by apluquet** [mitsio-motu-data/3D-Cartography-Drone#6]
Hello @Adrien-ANTON-LUDWIG,

I hear you've recently discovered a wrapping tool for markdown. Could you please use it on the entire mdbook in order to have a nice source code.

Thanks in advance! :grin:

**Comment by Adrien-LUDWIG** [comment]
Indeed, the tool is [`markdownlint`](https://github.com/DavidAnson/markdownlint). :rocket: 

However, I think it would be better to correct the formatting gradually, along with other upcoming changes.
That way we limit the impact on the Git history. :wink:
